### ISSUE TITLE
Project organizations

### ISSUE THREAD

**Issue by gaow** [bioworkflows/bioworkflows-help#23]
@BoPeng As discussed offline I'm trying to come up with the best approach to share and allow for collaborations for various workflows developed.

It is the most natural to have one workflow per github repository and also per bioworkflows repository. Then if i allow for contributions people can just go directly to the github repo and fork/work from there. The only drawback with this approach is that I have to make every workflow (typically just one ipynb file) a separate repository.

Another idea is I have a mega repo say `gaow/bioworkflows` and have different folders in it to create different bioworkflows repos, eg `gaow/bioworkflows/gwas` on my github will become `gaow/gwas` or whatever name on bioworkflows.com. But then the problem is when people want to contribute to just one file they have to also keep track of everything else going on in that repo, if they are directed to my github `gaow/bioworkflows` repo.

I wonder if there is a way bioworkflows.com can handle it better -- that is, somehow people can focus on bioworkflows.com, eg "fork" my bioworkflows repo which contains just the folder I created the repo with. Then send "pull request" after they make changes, but my `gaow/bioworkflows` mega repo on github will actually receive that change under `gaow/bioworkflows/gwas`. I can review on github and merge, and bioworkflows.com will sync it after i merge.

This is just one proposed mechanism, if we go for the mega repo on github organization. I"m open to other ways to organize.

**Comment by gaow** [comment]
@BoPeng A related issue to this, as we also have discussed offline, is that currently it seems to only display one pipeline from my upload that contains more than one pipelines: https://bioworkflows.com/gaow/GWAS/ 

No rush, I think it would be nice to ask my other colleagues to join after we iron out obvious issues like this. But I can ask my own teammates to test things out.

**Comment by BoPeng** [comment]
Yes, I adjusted the pipeline to generate preview for all notebooks, but the frontend lacks behind and only allow one landing page per workflow.
